test/test.js

```diff
@@ -0,0 +1,55 @@
+const { createS3UploadParams, reverse } = require("../index");
+const assert = require("assert");
+
+describe("Reverse", function () {
+  it(`reverses abc correctly`, function () {
+    assert.equal(reverse("abc"), "cba");
+  });
+  it("reverses abc and removes trailing spaces", function () {
+    assert.equal(reverse("abc "), "cba");
+  });
+  it("reverses abc and removes leading spaces", function () {
+    assert.equal(reverse(" abc"), "cba");
+  });
+  it("reverses abc and keeps spaces between non-space characters", function () {
+    assert.equal(reverse("ab c"), "c ba");
+  });
+});
+
+describe("CreateS3UploadParams", function () {
+  const ip_ranges_example = {
+    syncToken: "0123456789",
+    createDate: "yyyy-mm-dd-hh-mm-ss",
+    prefixes: [
+      {
+        ip_prefix: "cidr",
+        region: "region",
+        network_border_group: "network_border_group",
+        service: "subset",
+      }
```

Comment:
Codacy has a fix for the issue: [Replace `············}` with `······},`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
      },
```